compiler/testData/codegen/box/delegatedProperty/delegateToConst/delegateToConst.txt

```diff
@@ -0,0 +1,10 @@
+@kotlin.Metadata
```

Comment:
Regarding all `.txt` dumps: for historical reasons, they correspond to the old JVM backend's behavior by default. In case the output differs for JVM IR, the dump should be located in the file named e.g. `delegateToConst_ir.txt`. Please run `BlackBoxCodegenTestGenerated` (tests for the old backend) to output the dumps for the old backend, then manually create the `_ir.txt` file for each test, and rerun `IrBlackBoxCodegenTestGenerated` to update the dump for JVM IR.